[functional-tests] Method not allowed exception when tested with AWS S3

```
Test: presignedPostPolicy(PostPolicy policy)
java.lang.Exception: failed to upload object. Response: Response{protocol=http/1.1, code=405, message=Method Not Allowed, url=https://minio-java-test-3eec4n4.s3.amazonaws.com/minio-java-test-1lqo75c}, Error: <?xml version="1.0" encoding="UTF-8"?>
<Error><Code>MethodNotAllowed</Code><Message>The specified method is not allowed against this resource.</Message><Method>POST</Method><ResourceType>OBJECT</ResourceType><RequestId>3E1C91ACFEEEF668</RequestId><HostId>/DbM+t9s+9f5SHZJ8z9OCKQUP8CHyugcsw5WBdjgBmf4qsTdDsZ5zCJN5DEu5XeMQNfE8djYTA4=</HostId></Error>
        at FunctionalTest.presignedPostPolicy_test(FunctionalTest.java:1650)
        at FunctionalTest.runQuickTests(FunctionalTest.java:2552)
        at FunctionalTest.main(FunctionalTest.java:2609)
:functional:runFunctionalTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':functional:runFunctionalTest'.
> Process 'command '/usr/lib/jvm/java-8-openjdk-amd64/bin/java'' finished with non-zero exit value 255

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 1 mins 8.915 secs
```

--- a/functional/FunctionalTest.java
+++ b/functional/FunctionalTest.java
@@ -1630,7 +1630,7 @@ public class FunctionalTest {
       is.close();
 
       Request.Builder requestBuilder = new Request.Builder();
-      String urlString = client.getObjectUrl(bucketName, objectName);
+      String urlString = client.getObjectUrl(bucketName, "");
       Request request = requestBuilder.url(urlString).post(multipartBuilder.build()).build();
       OkHttpClient transport = new OkHttpClient();
       Response response = transport.newCall(request).execute();

